apps/coscrad-frontend/src/store/slices/utils/buildInitialLoadableState.ts

```diff
@@ -0,0 +1,7 @@
+import { ILoadable } from '../interfaces/loadable.interface';
+
+export const buildInitialLoadableState = <TData>(): ILoadable<TData> => ({
+    data: null,
```

Comment:
This is probably fine, but did we consider having a custom symbol to represent 'no load yet initiated'?